flocker/node/_change.py

```diff
@@ -62,16 +62,33 @@ def run_state_change(change, deployer):
 
     :return: ``Deferred`` firing when the change is done.
     """
+    if isinstance(change, _InParallel):
+        return gather_deferreds(list(
```

Comment:
I see that `gather_deferreds` was used in the previous implementation too but I forgot that I wrote that and I forgot that it exists in order to log all failures while returning only the first failure.

Perhaps that's no longer necessary if the error logging is going to be done with Eliot,  and closer to the constituent deferred.